packages/components/src/theme/ComposerTheme/Components/Skeleton.ts

```diff
@@ -0,0 +1,8 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+
+export const Skeleton: ComponentStyleConfig = {
+  baseStyle: {
+    '--skeleton-start-color': 'var(--chakra-colors-background-skeleton-start)',
+    '--skeleton-end-color': 'var(--chakra-colors-background-skeleton-end)',
```

Comment:
we should define variables as [shown in here](https://chakra-ui.com/docs/components/skeleton/theming#adding-a-custom-variant) instead of hardcoding the names